android/app/src/main/java/org/jboss/aerogear/unifiedpush/helloworld/handler/NotificationBarMessageHandler.java

```diff
@@ -41,7 +41,7 @@ public NotificationBarMessageHandler() {
     }
 
     @Override
-    public void onMessage(Context context, Bundle bundle) {
+    public void onMessage(Bundle bundle) {
         this.context = context;
```

Comment:
As we don't pass in the `Context` anymore - how do we get the Context ? 

I am getting NPE when the app receives a notification, while in background:
```
Process: org.jboss.aerogear.unifiedpush.helloworld, PID: 25071
                                                                                           java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.Context android.content.Context.getApplicationContext()' on a null object reference
                                                                                               at org.jboss.aerogear.unifiedpush.helloworld.handler.NotificationBarMessageHandler.onMessage(NotificationBarMessageHandler.java:49)
                                                                                               at org.jboss.aerogear.android.unifiedpush.RegistrarManager$3.run(RegistrarManager.java:183)
                                                                                               at java.lang.Thread.run(Thread.java:818)
```